server/src/main/java/com/cloud/network/router/CommandSetupHelper.java

```diff
@@ -728,6 +741,13 @@ public int compare(final PublicIpAddress o1, final PublicIpAddress o2) {
             final DataCenterVO dcVo = _dcDao.findById(router.getDataCenterId());
             cmd.setAccessDetail(NetworkElementCommand.ZONE_NETWORK_TYPE, dcVo.getNetworkType().toString());
 
+            Boolean lastIp = vlanLastIp.get(vlanTagKey);
+            if (lastIp == null) {
+                cmd.setAccessDetail(NetworkElementCommand.NETWORK_PUB_LAST_IP, "true");
+            } else {
+                cmd.setAccessDetail(NetworkElementCommand.NETWORK_PUB_LAST_IP, "false");
+            }
+
```

Comment:
here useAccessDetailsAccordingToVlanLastIpForGuestNetwork(); (long names just to be descriptive